WAIVER

```diff
@@ -1,6 +1,10 @@
-Technische Universiteit Delft hereby disclaims all copyright interest in the program “Citizen-Voice” (an inclusive, web-based software platform for collaborative data collection that facilitates citizen participation) written by the Author(s). 
-Prof. Aukje Hassoldt, Dean of Faculty of Technology, Policy and Management, and Prof.ir. Dick van Gameren, Faculty of Architecture and the Built Environment
+Technische Universiteit Delft hereby disclaims all copyright interest in the program “Citizen-Voice” (an inclusive, web-based 
+software platform for collaborative data collection that facilitates citizen participation) written by the Author(s). 
+Prof. Aukje Hassoldt, Dean of Faculty of Technology, Policy and Management, and Prof.ir. Dick van Gameren, Faculty of 
+Architecture and the Built Environment
 
-(c) 2022, Goncalves, Juliana E., Forgaci, Claudiu, and Verma Trivik. Citizen Voice is an initiative of the Centre for Urban Science & Policy at TU Delft. The project was funded by the Resilient Delta Initiative and received support from the Digital Competence Center of TU Delft., Delft, The Netherlands.
+(c) 2022, Goncalves, Juliana E., Forgaci, Claudiu, and Verma Trivik. Citizen Voice is an initiative of the Centre for Urban Science & 
```

Comment:
This is throwing a merge conflict for some reason. Maybe you could just pull this branch and push again to resolve that @cyasl ?

Reply:
sure!

Reply:
hey Gerdus, I think the version with line breaks is more recent. you can see on [this link](https://github.com/cyasl/Citizen-Voice/blob/devel/WAIVER) that they were recently added by Manuel. so I think it is better to resolve the conflict by accepting the version with the breaks instead of reverting back